teuthology/task/internal/__init__.py

```diff
@@ -334,7 +334,7 @@ def fetch_binaries_for_coredumps(path, remote):
                     log.error(e)
                     continue
             try:
-                dump_program = re.findall("from '([^']+)'", dump_out)[0]
+                dump_program = re.findall("from '([^ ']+)", dump_out)[0]
```

Comment:
or it is just missed `.split()[0]` at the end, however, for better readability maybe elaborate the code, like:
```
dump_command = re.findall("from '([^']+)", dump_out)[0]
dump_program = dump_command.split()[0]
```
If you like you could apply, or just approve the current changes.

Reply:
@kshtsk updated, thanks for the suggested improvement.